.circleci/config.yml

```diff
@@ -19,6 +27,29 @@ parameters:
         type: string
         default: wireit
 commands:
+    install-azcopy:
+        description: 'Install AzCopy and set up authentication'
+        steps:
+            - run:
+                  name: Install AzCopy
+                  command: |
+                      # Create a bin directory in the home folder
+                      mkdir -p ~/bin
+                      cd ~/bin
+                      wget -O azcopy.tar.gz https://github.com/Azure/azure-storage-azcopy/releases/download/v10.29.1/azcopy_linux_amd64_10.29.1.tar.gz
+                      tar -xf azcopy.tar.gz --strip-components=1
+                      chmod +x azcopy
+                      # Add to PATH
+                      echo 'export PATH=~/bin:$PATH' >> $BASH_ENV
+                      source $BASH_ENV
+                      azcopy --version
+
+                      # Set up Azure authentication
+                      echo 'export AZCOPY_AUTO_LOGIN_TYPE="SPN"' >> $BASH_ENV
+                      echo 'export AZCOPY_SPA_APPLICATION_ID="$AZURE_CLIENT_ID"' >> $BASH_ENV
+                      echo 'export AZCOPY_SPA_CLIENT_SECRET="$AZURE_CLIENT_SECRET"' >> $BASH_ENV
+                      echo 'export AZCOPY_TENANT_ID="$AZURE_TENANT_ID"' >> $BASH_ENV
+                      source $BASH_ENV
```

Comment:
Is this necessary? Looks like we’re already setting these in the node environment.